update the committee and constitution after the corresponding governance actions are enacted and add  column 'epoch' to table'committee_member'

Sotatek-HuyLe3a opened the pull request:
- Update the committee and constitution after the corresponding governance actions are enacted, write unit tests
- Add 'epoch' column to 'committee_member' table
- Update query for getting active committee members
- Add tx_index to table 'committee_registration/deregistration'

satran004 commented on stores/governance/src/main/java/com/bloxbean/cardano/yaci/store/governance/storage/impl/CommitteeMemberStorageImpl.java:
```diff
@@ -74,13 +89,14 @@ public List<CommitteeMemberDetails> getActiveCommitteeMembersDetailsByEpoch(int
                                                                 .and(COMMITTEE_DEREGISTRATION.CERT_INDEX.gt(COMMITTEE_REGISTRATION.CERT_INDEX))
```
Do you need to also check against tx_index and then cert_index? 
Looks like we don't have tx_index field in committee_registraion and committee_deregistration tables. I think we need to add that column to find the correct entry. You may want to refer to staking related tables.

Sotatek-HuyLe3a commented on stores/governance/src/main/java/com/bloxbean/cardano/yaci/store/governance/processor/CommitteeProcessor.java:
```diff
@@ -52,6 +64,31 @@ public void handleEpochChangeEvent(EpochChangeEvent epochChangeEvent) {
+                if (quorumThreshold != null) {
+                    BigInteger thresholdNumerator = null;
+                    BigInteger thresholdDenominator = null;
+                    Double threshold = quorumThreshold.doubleValue();
```
we should keep it as BigDecimal

Sotatek-HuyLe3a commented:
@satran004 , I added tx_index and updated the processors and queries accordingly. I also kept the committee threshold as a BigDecimal value. You can check it again.
